qiskit/synthesis/evolution/lie_trotter.py

```diff
@@ -100,26 +82,6 @@ def __init__(
         """
         super().__init__(1, reps, insert_barriers, cx_structure, atomic_evolution, wrap)
 
-    def synthesize(self, evolution):
```

Comment:
it's not deprecated - right? will be called as SuzukiTrotter.synthesize ?

Reply:
Correct 🙂